Makefile

```diff
@@ -32,6 +32,7 @@ go.mod:
 $(GO_DEPS): go.mod $(PATCHES)
 	# FIXME temporary workaround for crypto not downloading..
 	$(GO) get golang.org/x/crypto/ssh/terminal@e9b2fee46413
+	$(GO) get github.com/openconfig/gnmi@v0.0.0-20200617225440-d2b4e6a45802
```

Comment:
Fix has been already submitted https://github.com/Azure/sonic-telemetry/pull/64

Reply:
Thanks @pra-moh . I will close. I still need to understand why the go.mod file changes at all. It is supposed to be like a lock file I though, but somehow in the build process it changes.